app/styles/main.scss

```diff
@@ -20,6 +20,7 @@
 @import "./base/scrollable";
 @import "./base/advanced-filter";
 @import "./base/server-error";
+@import "./base/tree-component";
```

Comment:
Don't include that here anymore. Import it directly from the component and move the file to be in the same folder.